MapboxAndroidWearDemo/src/main/java/com/mapbox/mapboxandroiddemo/analytics/AnalyticsTracker.java

```diff
@@ -0,0 +1,177 @@
+package com.mapbox.mapboxandroiddemo.analytics;
+
+import android.os.Build;
+import android.support.annotation.NonNull;
+
+import com.segment.analytics.Analytics;
+import com.segment.analytics.messages.IdentifyMessage;
+import com.segment.analytics.messages.ScreenMessage;
+import com.segment.analytics.messages.TrackMessage;
+
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+/**
+ * This class abstracts various analytics calls to Segment analytics' Java library.
+ */
+
+public class AnalyticsTracker {
+
+  private static volatile AnalyticsTracker analyticsInstance;
+  private Analytics analytics = Analytics.builder("W72jV8mXRPrOkNPFh8Z9tfarSDTYIof4").build();
+
+  private static final String OPENED = "Opened app";
+  private static final String CLICKED_ON_SIGN_IN_BUTTON_EVENT_NAME = "Clicked on sign in button";
+  private static final String CLICKED_ON_CREATE_ACCOUNT_BUTTON_EVENT_NAME = "Clicked on create account button";
+  private static final String CLICKED_ON_NAV_DRAWER_SECTION_EVENT_NAME = "Clicked on nav drawer section";
+  private static final String CLICKED_ON_INDIVIDUAL_EXAMPLE_EVENT_NAME = "Clicked on individual example";
+  private static final String SECTION_NAME_MAP_KEY = "section name";
+  private static final String EXAMPLE_NAME_MAP_KEY = "example name";
+  private static final String IS_WEARABLE_MAP_VALUE = "wearable";
+  public static final String ORGANIZATION_NAME = "LangstonCompany";
```

Comment:
> Oh, this is the AnalyticsTracker class for the wearable app. Will need to update this entire class to basically match the non-wearable app.

All code that needs to be shared across modules (wear and non-wear) let's put in a separate new module (i.e. called `commons`, `shared`, `libs`) to avoid code dupes.